src/GoogleSheets/packages/auth/password/confirm/main.go

```diff
@@ -0,0 +1,69 @@
+package ConfirmResetPassword
+
+import (
+	"GoogleSheets/packages/common/Constants/AuthConstants"
+	"GoogleSheets/packages/common/Constants/SharedConstants"
+	"encoding/json"
+	"log"
+
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/cognitoidentityprovider"
+)
+
+func HandleRequest(ctx context.Context, event events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
+	var resetReq AuthConstants.ResetPassword
+	err := json.Unmarshal([]byte(event.Body), &resetReq)
+	if err != nil {
+		log.Printf(
+			"[ERROR] Auth - reset password json mismatch\nexpected: %v\nactual: %s",
+			AuthConstants.ResetPassword{Email: ".", Password: ".", Code: "."},
+			event.Body,
+		)
+		return events.APIGatewayProxyResponse{
+			StatusCode: 500,
+			Headers:    SharedConstants.ALLOW_ORIGINS_HEADER,
+			Body:       "Invalid request body",
+		}, nil
+	}
+
+	cfg, err := config.LoadDefaultConfig(ctx)
+	if err != nil {
+		fmt.Println("configuration error,", err)
+		return events.APIGatewayProxyResponse{
+			StatusCode: 500,
+			Headers:    SharedConstants.ALLOW_ORIGINS_HEADER,
+			Body:       "Internal server error",
+		}, nil
+	}
+
+	svc := cognitoidentityprovider.NewFromConfig(cfg)
+
+	input := &cognitoidentityprovider.ConfirmForgotPasswordInput{
+		ClientId:         aws.String(os.Getenv("COGNITO_CLIENT_ID")),
+		Username:         aws.String(resetReq.Email),
+		ConfirmationCode: aws.String(resetReq.Code),
+		Password:         aws.String(resetReq.Password),
+	}
+
+	_, err = svc.ConfirmForgotPassword(ctx, input)
```

Comment:
`err` can be many different errors, and we want to decipher whether its an actual system error (like the lambda failed) or if it's a user error like a code mismatch. If it's a user error, we want to return `400` status with a useful error message so it can be displayed on the frontend alert.

Please look through the SDK docs and figure out ones we should look for.
See [this as an example](https://github.com/jlui17/W2FHR/pull/6/files#diff-32e84a38a86b2ccd7198cc3e3865a7db9d3ca099c42be6b888a91c8253ab3459R73-R99).